packages/path_provider/path_provider/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
-## NEXT
+## 2.1.4
 
+* Adds Downloads directory support on Android
+* Updates minimum required path_provider_android to 2.2.0
+* Updates supported platforms and paths matrix to indicate that Downloads
+  directory now supported on Android.
```

Comment:
"... that Downloads is supported on Android."

Reply:
fixed